src/services/TransactionCommand.ts

```diff
@@ -223,4 +224,12 @@ export class TransactionCommand {
         }
         return undefined;
     }
+    protected createDeadline(deadlineInHours = 2): Deadline {
+        const deadline = Deadline.create(this.epochAdjustment, deadlineInHours);
+        const clientServerTimeDifference = AppStore.getters['network/clientServerTimeDifference'];
+        if (clientServerTimeDifference >= 0) {
+            return Deadline.createFromAdjustedValue(deadline.adjustedValue + clientServerTimeDifference);
+        }
+        return Deadline.createFromAdjustedValue(deadline.adjustedValue - clientServerTimeDifference);
```

Comment:
I think we don't need this check, just return it as below? If `clientServerTimeDifference`is negative it'll be a subtraction if positive then addition, right?

```
return Deadline.createFromAdjustedValue(deadline.adjustedValue + clientServerTimeDifference);
```